x/ccv/provider/keeper/proposal_test.go

```diff
@@ -525,28 +524,6 @@ func TestStopConsumerChain(t *testing.T) {
 			},
 			expErr: true,
 		},
-		{
-			description: "valid stop of consumer chain, throttle related queues are cleaned",
```

Comment:
I beleive the entire intention of this test case was to confirm the provider queues  are cleared. Since those queues no longer exist it seems neccessary to remove the whole test case